napari/components/_viewer/viewer.py

```diff
@@ -125,32 +108,9 @@ def active_markers(self, active_markers):
     def reset_view(self):
         """Resets the camera's view.
         """
-        self.camera.set_range()
+        self._qtviewer.view.camera.set_range()
 
-    def screenshot(self, region=None, size=None, bgcolor=None):
```

Comment:
Why did you move this to its Qt counterpart?

Reply:
The model should not know anything about Qt elements, much less about 'screenshots'.
This belongs to the view.